main.scss

```diff
@@ -0,0 +1,104 @@
+@import 'o-colors/main';
+@import 'o-spacing/main';
+@import 'o-typography/main';
+@import 'src/scss/variables';
+@import 'src/scss/functions';
+@import 'src/scss/mixins';
+
+/// Output All oMeter Features
+@mixin oMeter() {
+
+	:root {
+		--o-meter-width: 100%; // we need width to synch .o-meter-container with .o-meter-value to display the value label in the right position
+		--o-meter-height: 1em;
+	}
+
+	.o-meter {
+		display: block;
+		width: var(--o-meter-width);
+		height: var(--o-meter-height);
+		--o-meter-background-color: #{oColorsByName('slate-white-15')};
+		--o-meter-optimum-color: #{oColorsByName('jade')};
+		--o-meter-low-color: #ff8833; // 'mandarin' color used as hex because isn't an internal brand color
+		--o-meter-high-color: #{oColorsByName('crimson')}; // bad
+
+		/* Background in Firefox */
+		background: var(--o-meter-background-color);
+	}
+
+	.o-meter-container {
+		width: var(--o-meter-width);
+		position: relative;
+		.o-meter {
+			width: 100%;
+		}
+	}
+
+	.o-meter-value {
+		position: relative;
+		padding: 0 oSpacingByName('s1');
+		background: #{oColorsByName('black')};
+		color: #{oColorsByName('white')};
+		@include oTypographySans($scale: -1, $line-height: 24px);
```

Comment:
Our linting rules require a mixin to be used before other CSS properties are declared. We have this linting rule because it helps us be sure that the mixin is not going to properties are not going to overwrite the any of the other CSS properties in the rule block.

Reply:
thank you! got it!